src/components/TopBar/TopBar.vue

```diff
@@ -317,14 +353,22 @@ export default {
 			}
 		},
 
-		openSidebar() {
+		openSidebar(activeTab) {
+			if (typeof activeTab === 'string') {
+				emit('spreed:select-active-sidebar-tab', activeTab)
```

Comment:
This somehow does not set focus, right?
When I keyboard navigate to the participant icon and enter, the sidebar opens but the tab index is back on the call view?

Reply:
This is the case on master too, but only with firefox. I'll create an issue

Reply:
#8387